Funcky/Extensions/EnumerableExtensions/Shuffle.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+
+namespace Funcky.Extensions
+{
+    public static partial class EnumerableExtensions
+    {
+        /// <summary>
+        /// Returns the given sequence in random Order in O(n).
+        /// </summary>
```

Comment:
Add `<remarks>` specifying that the items are enumerated

Reply:
Addressed in 4718cf30dce6d5a47bf46e305baa318d928749b6